gather_th1_for_limits.py

```diff
@@ -14,12 +15,13 @@
 #jet_strings = ['j3', 'j4']
 jet_strings = ['j3b2', 'j3b3', 'j4b2', 'j4b3', 'j4b4']
 #training_strings = ['01', '02', '03', '04']
-training_strings = ['01']
+training_strings = ['0101010101'] #ver of j3b2+j3b3+j4b2+j4b3+j4b4
```

Comment:
Could you comment a bit more? Do you have a training with all jet categories together?

Reply:
Trained separately, but histograms are drawn in one code. Here, 0101010101 means the version of BDT classifiers in order of j3b2, j3b3, j4b2, j4b3, j4b4.